functests/utils/nodes/nodes.go

```diff
@@ -32,6 +33,20 @@ func GetByRole(c client.Client, role string) ([]corev1.Node, error) {
 	return nodes.Items, nil
 }
 
+// GetNonRTWorkers returns list of nodes with no worker-rt label
+func GetNonRTWorkers() ([]corev1.Node, error) {
+	nonRTWorkerNode := []corev1.Node{}
+
+	workerNodes, err := GetByRole(testclient.Client, "worker")
```

Comment:
please use a new const for it, like `testutils.RoleWorkerRT` (just without the need that it can be overridden by an env var, I think `worker` is always `worker` :) )